pkgs/by-name/mp/mpdris2/package.nix

```
@@ -40,21 +40,28 @@ python3.pkgs.buildPythonApplication rec {
     libnotify
   ];
 
-  propagatedBuildInputs = with python3.pkgs; [
+  propagatedBuildInputs = with python3Packages; [
```

```suggestion
  dependencies = with python3Packages; [
```
Didn't catch this one before.

---

even though it's `format = "other"`?

---

yes.

---

mkay, this does work, pushing